packages/smooth_app/lib/pages/preferences/user_preferences_page.dart

```diff
@@ -165,6 +167,38 @@ class _UserPreferencesPageState extends State<UserPreferencesPage>
               ),
             ),
           ),
+          SliverToBoxAdapter(
+            child: Column(
+              children: <Widget>[
+                getUserPreferences(
+                  type: PreferencePageType.ACCOUNT,
+                  userPreferences: userPreferences,
+                ).getOnlyHeader(),
+                SmoothSimpleButton(
+                  padding: const EdgeInsets.symmetric(
+                      horizontal: 30.0, vertical: 15.0),
```

Comment:
Either remove that padding or use app constants like `SMALL_SPACE`.

Reply:
ok